nwb_explorer/nwb_model_interpreter/nwb_model_interpreter.py

```diff
@@ -70,7 +75,20 @@ def assign_name_to_types(self, pynwb_obj):
     def add_variables_to_type(self, pynwb_obj, geppetto_composite_type, geppetto_common_library_access):
         ''' Use this function to add variables to a geppetto compositeType '''
         
-        if isinstance(pynwb_obj, TimeSeries):
+        if isinstance(pynwb_obj, ImageSeries):
+            # TODO lazy fetching with importValue
```

Comment:
Aren't we already using importValue for the timestamps?

Reply:
yes. Forgot to delete comment